src/sections/app.style.js

```diff
@@ -3,7 +3,7 @@ import { createGlobalStyle } from "styled-components";
 const GlobalStyle = createGlobalStyle`
 
 ::selection {
-    background: rgba(0, 70, 0, 0.99);
+    background: rgb(0, 179, 159);
```

Comment:
@thisiskaransgitOh I see now, see the preview it is wrong. Since this file is global style the color will go directly. You can add the Keppel color directly again.